pkg/util/chunking.go

```diff
@@ -1,7 +1,18 @@
 package util
 
+import (
+	"fmt"
+
+	"github.com/authzed/spicedb/internal/logging"
+)
+
 // ForEachChunk executes the given handler for each chunk of items in the slice.
 func ForEachChunk[T any](data []T, chunkSize uint64, handler func(items []T)) {
+	if chunkSize == 0 {
+		logging.Err(fmt.Errorf("ForEachChunk was given an invalid chunk size: %d", chunkSize))
```

Comment:
hrm, I'm not sure this message will be `sent` because it's missing `Msg` call or `Send` call. Can you double check?

Reply:
Lint caught it; fixed now